parm/use_cases/model_applications/s2s/UserScript_obsERA_obsOnly_Blocking/save_input_files_txt_missing.py

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env python
```

Comment:
Is this script still needed? If the logic here was replaced by the UserScript file list functionality, we should remove this script.